sydent/replication/pusher.py

```diff
@@ -32,6 +32,7 @@
 # Maximum amount of signed objects to replicate to a peer at a time
 EPHEMERAL_PUBLIC_KEYS_PUSH_LIMIT = 100
 INVITE_TOKENS_PUSH_LIMIT = 100
+INVITE_UPDATES_PUSH_LIMIT = 100
 ASSOCIATIONS_PUSH_LIMIT = 100
```

Comment:
Ideally we'd consolidate these with those in `replication.py` somehow, but feel free to ignore that for now.

Reply:
We do... https://github.com/matrix-org/sydent/blob/3de039dbac14dd69d57eac8b84cdbd3a9d5c8a82/sydent/http/servlets/replication.py#L38